tests/integration/security/mtls_first_party_jwt/strict_test.go

```diff
@@ -33,6 +33,7 @@ import (
 // - Send HTTP/gRPC requests between apps.
 func TestMtlsStrictK8sCA(t *testing.T) {
 	framework.NewTest(t).
+		Features("security.control-plane.mtls", "security.control-plane.k8s-certs").
```

Comment:
Not sure how this is "control-plane mtls". Can you add comments to the features yaml to explain what each one means?

Reply:
Will move it under an ingress sub directory